library/src/main/java/com/getstream/sdk/chat/adapter/BaseMessageListItemViewHolder.kt

```diff
@@ -11,18 +11,10 @@ abstract class BaseMessageListItemViewHolder<T : MessageListItem>(
     /**
      * Workaround to allow a downcast of the MessageListItem to T
      */
-    fun bindListItem(
-        messageListItem: MessageListItem,
-        position: Int
-    ) {
-        @Suppress("UNCHECKED_CAST")
-        bind(messageListItem as T, position)
-    }
+    @Suppress("UNCHECKED_CAST")
+    fun bindListItem(messageListItem: MessageListItem) = bind(messageListItem as T)
 
-    protected abstract fun bind(
-        messageListItem: T,
-        position: Int
-    )
+    protected abstract fun bind(messageListItem: T)
```

Comment:
At the end it was possible!!!! :rocket: :partying_face:

Reply:
It's very satisfying to finally have just the single parameter in this `bind` method, that's for sure 😄